groups/models.py

```diff
@@ -161,6 +161,20 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def subscribers(self):
```

Comment:
Well, it needs to know a bunch of related stuff to that comment - a discussion and a group - and all the users subscribed to those things.  That's quite a lot of information if you want to look at it from that angle.

I don't find the "shouldn't need to be aware of implementation details" angle convincing.  Where's the "should"?  What's the reason?  Code has to go somewhere, and personally I'd rather have the code to do a certain thing all in as contiguous a place as possible.  It makes comprehension easier (only reading one file), it makes debugging and updating easier (no need to swap back and forth between files), it makes testing easier (all your tests for this flow can be near or next to one another) and so on.  I need an actual reason rather than a "should".